lib/rules/block-indentation.js

```diff
@@ -161,17 +175,33 @@ module.exports = class BlockIndentation extends Rule {
 
   visitor() {
     this._elementStack = [];
+    this.seen = new Set();
+    this.nodeToSource = new WeakMap();
 
     return {
+      Template(node) {
+        if (this.mode === 'fix') {
+          if (
+            node.body[0] &&
+            node.body[0].type === 'TextNode' &&
+            !node.body[0].chars.replace(/ +/, '')
```

Comment:
Can you extract a helper function for this `isFoo(str)` to make it more clear what this is testing?